odoo/addons/test_http/tests/test_http.py

```diff
@@ -132,24 +156,57 @@ def test_static2_not_found(self):
         res = self.nodb_url_open("/test_http/static/i-dont-exist")
         self.assertEqual(res.status_code, 404)
 
-    def test_static3_attachment(self):
-        with file_open('test_http/static/src/img/gizeh.svg', 'rb') as file:
-            content = file.read()
-
-        attachment = self.env['ir.attachment'].create({
-            'name': 'point_of_origin.svg',
-            'type': 'binary',
-            'raw': content,
-            'res_model': 'test_http.stargate',
-            'res_id': self.ref('test_http.earth'),
-        })
-        attachment['url'] = f'/test_http/{attachment["checksum"]}'
+    def test_static3_attachment_fallback(self):
+        attachment = self.env.ref('test_http.one_pixel_png')
 
         res = self.db_url_open(attachment['url'])
-        self.assertEqual(res.headers.get('Content-Length'), '1529')
-        self.assertEqual(res.headers.get('Content-Type'), 'image/svg+xml; charset=utf-8')
+        res.raise_for_status()
+        self.assertEqual(res.headers.get('Content-Length'), '70')
+        self.assertEqual(res.headers.get('Content-Type'), 'image/png')
         self.assertEqual(res.headers.get('Content-Security-Policy'), "default-src 'none'")
-        self.assertEqual(res.content, content)
+        self.assertEqual(res.content, attachment.raw)
+
+    def test_static4_web_content(self):
```

Comment:
Along with `test_static5_web_image`, not strictly part of this PR but still nice to have. I plan to enrich `/web/content` and `/web/image` with X-Accel capabilities in the coming weeks.